src/lib/utils/md.ts

```diff
@@ -67,12 +72,14 @@ const removeAnchorLinks = (mdContent: string) =>
 
 export const getContentBySlug = (slug: string, fields: string[] = []) => {
   const realSlug = `${slug}/index.md`
-  const fullPath = join(contentDir, realSlug)
+  const fullPath = join(CURRENT_CONTENT_DIR, realSlug)
   const fileContents = fs.readFileSync(fullPath, "utf8")
   const { data: frontmatter, content } = matter(fileContents)
 
   type Items = {
-    [key: string]: string | Frontmatter
+    slug?: string
```

Comment:
`slug` and `content` shouldn't be optional properties here, they are required to build any content page. Not sure about `frontmatter`

Reply:
Fyi, #33 also introduces `tocItems` here, since it parsed the content before the custom-header-ids are stripped out for rendering.

As such, encountered this myself and had played around with a similar fix... would support being declarative about the exact properties and types we want here.

Reply:
@pettinarip `[key: string]: string | Frontmatter` doesn't imply that all the possible fields could be of type `Frontmatter`, it's a union type, meaning it can be one of both.

Reply:
@pettinarip I'm ok with using these fixed fields if we're sure this works with all files, as we don't have the same fields on all markdown files AFAIK.

Reply:
Agreed. I think these 3 fields are going to be used by all the pages we have atm.

Pushed a new commit with those changes https://github.com/ethereum/ethereum-org-fork/pull/30/commits/11334302c22d069fac3e9559e751859dae043e35